packages/app/app/actions/search.js

```diff
@@ -298,3 +300,25 @@ export const albumInfoSearchByName = (albumName, history) => async (dispatch, ge
     ));
   }
 };
+
```

Comment:
I have a suggestion about these actions that will make these more concise:

```javascript
export const setSearchDropdownVisibility =  displayState => ({
  type: EARCH_DROPDOWN_DISPLAY_CHANGE,
  payload: displayState
})


```

No need to use a thunk (the async function with dispatch) since it's not a complex action.